invenio/values.yaml

```diff
@@ -9,7 +9,6 @@ volume:
     size: 10Gi
 nginx:
   max_conns: 100
```

Comment:
IMO it still makes sense to limit the number of connections reaching the application to the one that it can concurrently handle.

Reply:
Well it's in the same Pod but it is a different container, also filebeat is in the same pod. Summing up their values to one could be confusing. Maybe we could consider making a bigger tree and including these 3 ones under `web` for instance and renaming current `web` to `application`.

Reply:
> IMO it still makes sense to limit the number of connections reaching the application to the one that it can concurrently handle.

Definetly, my point was only about moving things of place. But yes, since we alo have filebeat, lets not overcomplicate the tree :)

Resolving...